lhotse/dataset/vad.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+from math import isclose
+from typing import Dict, Optional
+
+import numpy as np
+import torch
+from torch.utils.data import Dataset
+
+from lhotse.cut import CutSet
+from lhotse.utils import Pathlike, Seconds
+
+EPS = 1e-8
+
+
+class VadDataset(Dataset):
```

Comment:
Can we have a unit test that builds the `VadDataset` on a single-element `CutSet`, obtains `dataset[0]` and checks that the `is_voice` field contains correct supervision (e.g. 0-2s silence, 2-5s speech)?

Reply:
(actually, it's best if that cut has two supervisions and silence between them - that tests the supervision iteration in __getitem__)